admission-plugin.yaml

```diff
@@ -0,0 +1,7 @@
+admissionConfig:
```

Comment:
remove this file, move to doc or PR description?

Reply:
> remove this file, move to doc or PR description?

Yep, to be removed. Was here for my dev convenience. 

I have a few unit tests to complete then I'll split the PR up into more discrete commits which should make it easier to review.